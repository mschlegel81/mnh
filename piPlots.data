[[[10,0.099753034660391],[20,0.0499688469219546],[30,0.0333240868908471],[40,0.0249960967959609],[50,0.0199980009987828],[60,0.0166655096607969],[70,0.0142849856085594],[80,0.012499511814072],[90,0.0111107682284852],[100,0.00999975003124047],[110,0.00909072128160777],[120,0.00833318866996342],[130,0.0076921939091888],[140,0.00714276604079497],[150,0.00666659259670726],[160,0.00624993896782363],[170,0.0058823020579711],[180,0.0055555126902682],[190,0.00526312144753693],[200,0.00499996875097649],[210,0.00476187776774492],[220,0.00454543106747335],[230,0.00434780554005975],[240,0.00416664858256836],[250,0.00399998400031998],[260,0.00384613962247511],[270,0.00370369100260562],[280,0.00357141718312618],[290,0.00344826561169854],[300,0.00333332407420267],[310,0.00322579805992587],[320,0.0031249923706986],[330,0.00303029607376435],[340,0.00294117010998125],[350,0.00285713702629856],[360,0.00277777241946183],[370,0.00270269776720594],[380,0.00263157439135013],[390,0.00256409834963594],[400,0.00249999609378052],[410,0.00243902076292943],[420,0.00238094900661067],[430,0.00232557825099287],[440,0.00227272433792278],[450,0.00222221947875493],[460,0.00217391047507065],[470,0.00212765716653742],[480,0.00208333107278426],[490,0.00204081420157674],[500,0.00199999800001],[510,0.00196078242908988],[520,0.00192307514509243],[530,0.0018867907735991],[540,0.00185185026419418],[550,0.00181818031555843],[560,0.00178571286215946],[570,0.00175438461497444],[580,0.0017241366497239],[590,0.00169491403697991],[600,0.00166666550926328],[610,0.00163934316088501],[620,0.00161290217683534],[630,0.0015873005874927],[640,0.00156249904632859],[650,0.00153846062813196],[660,0.00151515064557669],[670,0.00149253648221588],[680,0.0014705874402118],[690,0.00144927460130668],[700,0.00142857069971031],[710,0.00140845000572941],[720,0.00138888821909455],[730,0.0013698623710547],[740,0.00135135073441003],[750,0.00133333274074206],[760,0.00131578890417823],[770,0.00129870075109691],[780,0.00128205075523971],[790,0.00126582227775186],[800,0.0012499995117197],[810,0.00123456743081636],[820,0.00121951174170511],[830,0.00120481883988348],[840,0.00119047576868123],[850,0.00117647018115275],[860,0.00116279030462793],[870,0.00114942490770796],[880,0.0011363632695113],[890,0.00112359515099301],[900,0.00111111076817611],[910,0.00109890076714786],[920,0.00108695620068676],[930,0.00107526850639748],[940,0.00106382948624143],[950,0.00105263128736008],[960,0.00104166638409688],[970,0.00103092756113124],[980,0.00102040789764503],[990,0.0010101007524488],[1000,0.000999999750000312]],[[1,1.14159265358979],[2,0.474925986923127],[3,0.20825932025646],[4,0.0939736059707456],[5,0.0431799551770948],[6,0.0200919320890717],[7,0.00943592143306108],[8,0.00446311646025611],[9,0.002122972943642],[10,0.00101448390945638],[11,0.0004866319884156],[12,0.000234181069656969],[13,0.000113004628652825],[14,0.0000546604163174972],[15,0.0000264942448452697],[16,0.0000128654521974175],[17,6.25755273179172E-6],[18,3.04800156277385E-6],[19,1.48659829135979E-6],[20,7.25914646312062E-7],[21,3.54849453605482E-7],[22,1.73631568795756E-7],[23,8.50361584437895E-8],[24,4.16809576334835E-8],[25,2.04457572367759E-8],[26,1.00363452780732E-8],[27,4.92984129755493E-9],[28,2.4230120711493E-9],[29,1.19158718786191E-9],[30,5.86310550457214E-10],[31,2.88633515959508E-10],[32,1.42157514492619E-10],[33,7.00462520018125E-11],[34,3.4528764829983E-11],[35,1.7027394685204E-11],[36,8.39995781265479E-12],[37,4.14533186422761E-12],[38,2.04638281146863E-12],[39,1.01053822428032E-12],[40,4.9917166754232E-13],[41,2.46644935378293E-13],[42,1.21903572306015E-13],[43,6.02664619597792E-14],[44,2.98016819555436E-14],[45,1.47399453753749E-14],[46,7.29191013126851E-15],[47,3.60887535133525E-15],[48,1.78503045678013E-15],[49,8.83841611010183E-16],[50,4.38451358553138E-16],[51,2.17274115366095E-16],[52,1.07552855510562E-16],[53,5.3776427755281E-17],[54,2.68882138776405E-17],[55,1.32272665043232E-17],[56,7.37257477290143E-18],[57,3.46944695195361E-18],[58,1.95156391047391E-18],[59,4.33680868994202E-19],[60,1.30104260698261E-18]],[[0.00012,0.099753034660391],[0.0002,0.0499688469219546],[0.0003,0.0333240868908471],[0.00048,0.0249960967959609],[0.00062,0.0199980009987828],[0.00067,0.0166655096607969],[0.00076,0.0142849856085594],[0.00086,0.012499511814072],[0.00089,0.0111107682284852],[0.00079,0.00999975003124047],[0.00089,0.00909072128160777],[0.00089,0.00833318866996342],[0.00102,0.0076921939091888],[0.00106,0.00714276604079497],[0.00115,0.00666659259670726],[0.00123,0.00624993896782363],[0.00133,0.0058823020579711],[0.00141,0.0055555126902682],[0.00151,0.00526312144753693],[0.00157,0.00499996875097649],[0.00167,0.00476187776774492],[0.00175,0.00454543106747335],[0.002,0.00434780554005975],[0.00208,0.00416664858256836],[0.0021,0.00399998400031998],[0.00209,0.00384613962247511],[0.00221,0.00370369100260562],[0.00223,0.00357141718312618],[0.00232,0.00344826561169854],[0.00243,0.00333332407420267],[0.0025,0.00322579805992587],[0.00259,0.0031249923706986],[0.00271,0.00303029607376435],[0.00299,0.00294117010998125],[0.00283,0.00285713702629856],[0.00294,0.00277777241946183],[0.00301,0.00270269776720594],[0.0031,0.00263157439135013],[0.0032,0.00256409834963594],[0.00328,0.00249999609378052],[0.00343,0.00243902076292943],[0.00348,0.00238094900661067],[0.00359,0.00232557825099287],[0.00368,0.00227272433792278],[0.00376,0.00222221947875493],[0.00387,0.00217391047507065],[0.00393,0.00212765716653742],[0.00398,0.00208333107278426],[0.00409,0.00204081420157674],[0.00415,0.00199999800001],[0.00426,0.00196078242908988],[0.0044,0.00192307514509243],[0.00443,0.0018867907735991],[0.00456,0.00185185026419418],[0.00464,0.00181818031555843],[0.00472,0.00178571286215946],[0.00486,0.00175438461497444],[0.00499,0.0017241366497239],[0.00515,0.00169491403697991],[0.0051,0.00166666550926328],[0.0052,0.00163934316088501],[0.00526,0.00161290217683534],[0.00541,0.0015873005874927],[0.00548,0.00156249904632859],[0.0056,0.00153846062813196],[0.00592,0.00151515064557669],[0.00636,0.00149253648221588],[0.00584,0.0014705874402118],[0.00599,0.00144927460130668],[0.00606,0.00142857069971031],[0.00613,0.00140845000572941],[0.00628,0.00138888821909455],[0.00635,0.0013698623710547],[0.00644,0.00135135073441003],[0.0066,0.00133333274074206],[0.00669,0.00131578890417823],[0.00669,0.00129870075109691],[0.00668,0.00128205075523971],[0.00694,0.00126582227775186],[0.00704,0.0012499995117197],[0.00714,0.00123456743081636],[0.00724,0.00121951174170511],[0.00725,0.00120481883988348],[0.00746,0.00119047576868123],[0.00757,0.00117647018115275],[0.00757,0.00116279030462793],[0.00767,0.00114942490770796],[0.00769,0.0011363632695113],[0.00792,0.00112359515099301],[0.00792,0.00111111076817611],[0.00817,0.00109890076714786],[0.00803,0.00108695620068676],[0.00831,0.00107526850639748],[0.00844,0.00106382948624143],[0.00831,0.00105263128736008],[0.00858,0.00104166638409688],[0.00873,0.00103092756113124],[0.00902,0.00102040789764503],[0.00873,0.0010101007524488],[0.0092,0.000999999750000312]],[[0.0002,1.14159265358979],[0.00027,0.474925986923127],[0.00033,0.20825932025646],[0.00042,0.0939736059707456],[0.00048,0.0431799551770948],[0.00056,0.0200919320890717],[0.00067,0.00943592143306108],[0.00076,0.00446311646025611],[0.00087,0.002122972943642],[0.00099,0.00101448390945638],[0.00145,0.0004866319884156],[0.00162,0.000234181069656969],[0.00153,0.000113004628652825],[0.00147,0.0000546604163174972],[0.00156,0.0000264942448452697],[0.00173,0.0000128654521974175],[0.00251,6.25755273179172E-6],[0.00222,3.04800156277385E-6],[0.00219,1.48659829135979E-6],[0.00233,7.25914646312062E-7],[0.00251,3.54849453605482E-7],[0.00274,1.73631568795756E-7],[0.00286,8.50361584437895E-8],[0.00303,4.16809576334835E-8],[0.00321,2.04457572367759E-8],[0.00343,1.00363452780732E-8],[0.00358,4.92984129755493E-9],[0.00376,2.4230120711493E-9],[0.00406,1.19158718786191E-9],[0.00421,5.86310550457214E-10],[0.00448,2.88633515959508E-10],[0.00472,1.42157514492619E-10],[0.00495,7.00462520018125E-11],[0.00515,3.4528764829983E-11],[0.00541,1.7027394685204E-11],[0.00566,8.39995781265479E-12],[0.00613,4.14533186422761E-12],[0.00613,2.04638281146863E-12],[0.00644,1.01053822428032E-12],[0.00676,4.9917166754232E-13],[0.00696,2.46644935378293E-13],[0.00736,1.21903572306015E-13],[0.00757,6.02664619597792E-14],[0.0078,2.98016819555436E-14],[0.00829,1.47399453753749E-14],[0.00831,7.29191013126851E-15],[0.00873,3.60887535133525E-15],[0.00904,1.78503045678013E-15],[0.00954,8.83841611010183E-16],[0.0097,4.38451358553138E-16],[0.01096,2.17274115366095E-16],[0.0112,1.07552855510562E-16],[0.01096,5.3776427755281E-17],[0.01096,2.68882138776405E-17],[0.01142,1.32272665043232E-17],[0.0117,7.37257477290143E-18],[0.01226,3.46944695195361E-18],[0.01256,1.95156391047391E-18],[0.01287,4.33680868994202E-19],[0.01318,1.30104260698261E-18]]]