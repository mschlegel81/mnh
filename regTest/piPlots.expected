Row #1: series A, error vs. summand count
Row #2: series B, error vs. summand count
Row #3: series A, error vs. time
Row #4: series B, error vs. time
73971ECB2A35DB12
